image_view/src/nodes/image_view.cpp

```diff
@@ -120,13 +119,14 @@ static void mouseCb(int event, int x, int y, int flags, void* param)
     RCLCPP_WARN(node->get_logger(), "Couldn't save image, no data!");
     return;
   }
-
-  std::string filename = (g_filename_format % g_count).str();
+  sprintf(buf, format_string.c_str(), g_count);
+  std::string filename = buf;
   if (cv::imwrite(filename, image)) {
     RCLCPP_INFO(node->get_logger(), "Saved image %s", filename.c_str());
     g_count++;
   } else {
-    boost::filesystem::path full_path = boost::filesystem::complete(filename);
+    std::experimental::filesystem::path full_path = 
```

Comment:
This won't work on MacOS, I suggest looking at camera_info_manager/filesystem_helper.hpp

Reply:
I commented out this feature, and filesystem will add to macOS in the future, when it add to macOS, this function will enable ASAP :).